### ISSUE TITLE
Extraneous space in AWS theme element preventing its use

### ISSUE THREAD

**Issue by deinspanjer** [structurizr/structurizr#102]
Was trying to put an AWS API Gateway on a deployment diagram, and it took me a little digging to figure out why it wasn't working.
In the theme json file, there is an extraneous space before the word Amazon. I tried changing the tag name to match that, but the space is also reflected in the URL for the icon which also breaks it.

```{
    "tag" : "Amazon Web Services -  Amazon API Gateway",
    "stroke" : "#d72b6c",
    "color" : "#d72b6c",
    "icon" : "https://static.structurizr.com/themes/amazon-web-services-2022.04.30/Arch_ Amazon-API-Gateway_48.png"
  }```

**Comment by deinspanjer** [comment]
I've found two more icons that aren't working, but it doesn't appear that they are related to space issues. Should I open a new issue for them or do you want to tackle them in here?

Amazon Web Services - Simple Storage Service
Amazon Web Services - Simple Storage Service Bucket

Here is a quick change I made to the AWS Deployment example DSL that demonstrates it:
```
workspace "Amazon Web Services Example" "An example AWS deployment architecture." {

    model {
        springPetClinic = softwaresystem "Spring PetClinic" "Allows employees to view and manage information regarding the veterinarians, the clients, and their pets." "Spring Boot Application" {
            webApplication = container "Web Application" "Allows employees to view and manage information regarding the veterinarians, the clients, and their pets." "Java and Spring Boot"
            database = container "Database" "Stores information regarding the veterinarians, the clients, and their pets." "Relational database schema" "Database"
            files = container "File store" "Stores pictures of pets"
        }

        webApplication -> database "Reads from and writes to" "MySQL Protocol/SSL"
        webApplication -> files "Links to pictures" "HTTP"

        live = deploymentEnvironment "Live" {

            deploymentNode "Amazon Web Services" {
                tags "Amazon Web Services - Cloud"

                region = deploymentNode "US-East-1" {
                    tags "Amazon Web Services - Region"

                    route53 = infrastructureNode "Route 53" {
                        description "Highly available and scalable cloud DNS service."
                        tags "Amazon Web Services - Route 53"
                    }

                    elb = infrastructureNode "Elastic Load Balancer" {
                        description "Automatically distributes incoming application traffic."
                        tags "Amazon Web Services - Elastic Load Balancing"
                    }

                    deploymentNode "Autoscaling group" {
                        tags "Amazon Web Services - Auto Scaling"

                        deploymentNode "Amazon EC2" {
                            tags "Amazon Web Services - EC2"

                            webApplicationInstance = containerInstance webApplication
                        }
                    }
                    
                    s3 = infrastructureNode "AWS S3" {
                            description "Hosted File Server"
                            tags "Amazon Web Services - Simple Storage Service"
                    }
                    s3Bucket = containerInstance "files" {
                                description "S3 bucket containing picture files"
                                tags "Amazon Web Services - Simple Storage Service Bucket"

                                s3 -> this "serves files from" "REST"
                    }
                    deploymentNode "Amazon RDS" {
                        tags "Amazon Web Services - RDS"

                        deploymentNode "MySQL" {
                            tags "Amazon Web Services - RDS MySQL instance"

                            databaseInstance = containerInstance database
                        }
                    }

                }
            }

            route53 -> elb "Forwards requests to" "HTTPS"
            elb -> webApplicationInstance "Forwards requests to" "HTTPS"
        }
    }

    views {
        deployment springPetClinic "Live" "AmazonWebServicesDeployment" {
            include *
            autolayout lr

            animation {
                route53
                elb
                webApplicationInstance
                databaseInstance
            }
        }

        styles {
            element "Element" {
                shape roundedbox
                background #ffffff
            }
            element "Database" {
                shape cylinder
            }
            element "Infrastructure Node" {
                shape roundedbox
            }
        }

        themes https://static.structurizr.com/themes/amazon-web-services-2020.04.30/theme.json
    }

}```

**Comment by simonbrowndotje** [comment]
Thanks ... this should now be fixed.

**Comment by deinspanjer** [comment]
The URL for the icon is fixed which prevents a crash if you use the tag name with the double space, but I think it would be better to also remove the double space from the tag name.

While it is technically breaking backward compatibility, I don't think anyone else has been using that tag since the behavior before your change was that the entire diagram would fail to render.

**Comment by simonbrowndotje** [comment]
The tag name is also fixed ... there will be some caching happening on the JSON file, but it should refresh at some point.

**Comment by deinspanjer** [comment]
Okay, the correction isn't showing up in the structurizr/lite:latest docker image I was using for local dev for some reason, but when I went ahead and saved and pushed to the cloud, the styled tag does appear there properly.

I'm unsure why the docker lite run isn't working. I tried deleting the docker image and re-fetching it, but still the same result.
Regardless, it looks like the bug as reported has been fixed. Thanks!
